<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmAssayEdit</class>
 <widget class="QDialog" name="FrmAssayEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,9">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labTitle">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="verticalSpacing">
          <number>20</number>
         </property>
         <item row="0" column="1">
          <widget class="QLabel" name="label_29">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>化验编号:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="labAssayNum">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>粮食类型:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4" colspan="2">
          <widget class="QLabel" name="labGrainType">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string>样品级别:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QLabel" name="labClass">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_31">
           <property name="text">
            <string>样盒号码:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labSampleBox">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_32">
           <property name="text">
            <string>标准样:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4" colspan="2">
          <widget class="QLabel" name="labStandard">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string>化验完成:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="QLabel" name="labComplete">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_34">
           <property name="text">
            <string>是否拒收:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="labReject">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>异味:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QComboBox" name="cbxSmell">
           <item>
            <property name="text">
             <string>无</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>有</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="5">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>56</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>容重:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="editRZ"/>
         </item>
         <item row="3" column="3">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>水分:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4" colspan="2">
          <widget class="QLineEdit" name="editSF"/>
         </item>
         <item row="3" column="6">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>霉变:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="7">
          <widget class="QLineEdit" name="editMB"/>
         </item>
         <item row="4" column="0">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>破损:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLineEdit" name="editPS"/>
         </item>
         <item row="4" column="3">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>热损:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="4" colspan="2">
          <widget class="QLineEdit" name="editRS"/>
         </item>
         <item row="4" column="6">
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>并间杂:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="7">
          <widget class="QLineEdit" name="editBJZ"/>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>小石子:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLineEdit" name="editXSZ"/>
         </item>
         <item row="5" column="3">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>土块:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="4" colspan="2">
          <widget class="QLineEdit" name="editTK"/>
         </item>
         <item row="5" column="6">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>棒芯:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="7">
          <widget class="QLineEdit" name="editBX"/>
         </item>
         <item row="5" column="8">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>水泡粒:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLineEdit" name="editSPL"/>
         </item>
         <item row="6" column="3">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>异种粮:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="4" colspan="2">
          <widget class="QLineEdit" name="editYZL"/>
         </item>
         <item row="6" column="6">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>样品毛重:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="7">
          <widget class="QLineEdit" name="editMZ"/>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>样品净重:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLineEdit" name="editJZ"/>
         </item>
         <item row="7" column="3">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>杂质:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="4" colspan="2">
          <widget class="QLineEdit" name="editZZ">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="6">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>留样号码:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="7">
          <widget class="QLineEdit" name="editBugNUm"/>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>备注:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>拒收原因:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="2">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="8" column="2" colspan="6">
          <widget class="QLineEdit" name="editRemark"/>
         </item>
         <item row="9" column="2" colspan="6">
          <widget class="QLineEdit" name="editReject">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnEdit">
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnReturn">
          <property name="text">
           <string>返回</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
